src/databricks/labs/ucx/source_code/notebooks/loaders.py

```diff
@@ -52,7 +52,7 @@ def load_dependency(self, path_lookup: PathLookup, dependency: Dependency) -> So
             return None
         try:
             content = absolute_path.read_text("utf-8")
-        except NotFound:
+        except (NotFound, PermissionError):
             logger.warning(f"Could not read notebook from workspace: {absolute_path}")
```

Comment:
If we're going to suppress the underlying cause, I think I'd prefer to have separate and different logging to help with diagnosing why it didn't work.

```suggestion
        except NotFound:
            logger.warning(f"Path not found trying to read notebook from workspace: {absolute_path}")
        except PermissionError as e:
            logger.warning(f"Permissions error ({e}) while reading notebook from workspace: {absolute_path}")
```